docs/admin-manual/cm-configuration.rst

```diff
@@ -977,14 +1056,49 @@ daemon to the *condor_collector* daemon, it is possible for the limit
 to be exceeded. If the limits are exceeded, HTCondor will not kill any
 job to reduce the number of running jobs to meet the limit.
 
+Running Multiple Negotiators in One Pool
+''''''''''''''''''''''''''''''''''''''''
+
+Usually, a single HTCondor pool will have a single *condor_collector* instance
+running and a single *condor_negotiator* instance running.  However, there are
+special situation where you may want to run more than one *condor_negotiator*
+against a *condor_collector*, and still consider it one pool.
+
+In such a scenario, each *condor_negotiator* is responsible for some
+non-overlapping partition of the slots in the pool.  This might be for
+performance -- if you have more than 100,000 slots in the pool, you may need to
+shard this pool into several smaller sections in order to lower the time each
+negotiator spends.  Because accounting is done at the negotiator level, you
+may want to do this to have separate accounting and distinct fair share between
+different kinds of machines in your pool.  For example, let's say you have some
+GPU machines and non-GPU machines, and you want usage of the non-GPU machine to
+not "count" against the fair-share usage of GPU machines.  One way to do this
+would be to have a separate negotiator for the GPU machines vs the non-GPU
+machines.   At Wisconsin, we have a separate, small subset of our pool for
```

Comment:
```suggestion
machines.   At UW-Madison, we have a separate, small subset of our pool for
```
I feel like `At Wisconsin` is very broad... maybe this should even just be `At the CHTC` or something.